internal/resource/config/oauth/authserversettings/oauth_auth_server_settings_resource.go

```diff
@@ -89,7 +89,9 @@ func (r *oauthAuthServerSettingsResource) Schema(ctx context.Context, req resour
 		Attributes: map[string]schema.Attribute{
 			"default_scope_description": schema.StringAttribute{
 				Description: "The default scope description.",
-				Required:    true,
+				Optional:    true,
+				Computed:    true,
+				Default:     stringdefault.StaticString(""),
```

Comment:
Do we need a default here, or would it be fine to just leave this optional and not computed?

Reply:
The default value returned from PingFederate is an empty string.